meta-xt-domu/recipes-kernel/linux/linux-virtio-armv8.bb

```diff
@@ -0,0 +1,34 @@
+DESCRIPTION = "Virtio recipe for AArch64 kernels"
+
+inherit logging
```

Comment:
Are you using logging in this recipe?

Reply:
I have used it for debugging.
Will remove.

Reply:
Done